targets/frontend/src/modules/agreements/components/Edition/index.tsx

```diff
@@ -11,13 +11,12 @@ import React from "react";
 import { useAgreementUpdateMutation } from "./agreement.mutation";
 import { useAgreementQuery } from "./agreement.query";
 import { AgreementForm } from "../Common";
-import { usePublishMutation } from "./publish.mutation";
+import { usePublishMutation } from "../../../documents/components/publish.mutation";
```

Comment:
J'ai homogénéisé le usePublicationMutation pour en avoir un seul dans documents. Ca évite de devoir modifier X graphQL quand on veut modifier l'API...

Reply:
coool